src/admin/class-cmb2.php

```diff
@@ -227,4 +207,30 @@ public static function cmb2_render_openstreetmap_field_type( $field, $escaped_va
 
 		echo '<div id="map-base" class="map-base"></div>';
 	}
+
+	/**
+	 * Adds a custom field type for a button to sync or import data.
+	 *
+	 * @param  object $field             The CMB2_Field type object.
+	 * @param  mixed  $escaped_value     The value of this field passed through the escaping filter.
+	 * @param  int    $object_id         The ID of the object this field is saving to.
+	 *
+	 * @return void
+	 */
+	public function cmb2_render_import_sync_field_type( $field, $escaped_value, $object_id ) {
+		$last_updated = get_post_meta( $object_id, 'datalayer_last_import', true );
+
+		// Get URL of edit post page.
+		$edit_post_url = get_edit_post_link( $object_id );
+
+		echo '<form method="post" style="margin-top:10px;">
+                <input type="hidden" name="sync_import_file" value="1">
+                <input type="hidden" name="post_id" value="' . esc_attr( $object_id ) . '">
+                <input type="hidden" name="redirect_url" value="' . esc_attr( $edit_post_url ) . '">
+                <input type="submit" name="submit_sync_import_file" class="button button-primary button-large" value="' . esc_html__( 'Sync data', 'openkaarten-base' ) . '">
```

Comment:
Ik besef me hierdoor pas dat je de live datalayers ook importeert. Daar valt iets voor te zeggen, maar "live" suggereert wel dat er niet elke keer op een knop gedrukt hoeft te worden om te synchroniseren. Dus dan zou ik ook wel iets van een cron verwachten die de feed eens in de X tijd automatisch synchroniseert.

Edit: En verderop besef ik me dat je twee opties hebt: import en live. Zou import met cron wellicht nog een goede toevoeging zijn? Bij import is het immers wel mogelijk om markers en tooltips toe te voegen. (Voor het nice-to-have lijstje)